packages/react-examples/src/react/MessageBar/MessageBar.Basic.Example.tsx

```diff
@@ -184,32 +165,39 @@ export const MessageBarBasicExample: React.FunctionComponent = () => {
   const resetChoice = React.useCallback(() => setChoice(undefined), []);
 
   return (
-    <Stack {...horizontalStackProps}>
-      <StackItem disableShrink>
-        <ChoiceGroup
-          styles={choiceGroupStyles}
-          label="Select a MessageBar Example Below. To test in narrator, show one message at a time."
-          selectedKey={choice}
-          // eslint-disable-next-line react/jsx-no-bind
-          onChange={(e, v) => setChoice(v!.key)}
-          options={choiceOptions}
-        />
-      </StackItem>
-      <Stack {...verticalStackProps}>
-        {(choice === 'default' || showAll) && <DefaultExample />}
-
-        {(choice === 'error' || showAll) && <ErrorExample resetChoice={resetChoice} />}
-
-        {(choice === 'blocked' || showAll) && <BlockedExample resetChoice={resetChoice} />}
-
-        {(choice === 'severe' || showAll) && <SevereExample resetChoice={resetChoice} />}
-
-        {(choice === 'success' || showAll) && <SuccessExample />}
-
-        {(choice === 'warning' || showAll) && <WarningExample resetChoice={resetChoice} />}
-
-        {(choice === 'warning2' || showAll) && <WarningExample2 resetChoice={resetChoice} />}
+    <div>
+      <Text block>
+        Because screen readers will immediately read any MessageBar(s) on page load, the examples are hidden by default.
```

Comment:
We've gotten at least one issue saying "why aren't the examples shown by default, this is confusing" so I added an explanation, and moved the "show all" option to the front so it's more obvious if people want to see them all.